src/v2/conversions.rs

```diff
@@ -914,19 +947,51 @@ impl TryFrom<AccountInfo> for super::types::AccountInfo {
             index,
             stake,
             address,
+            cooldowns,
+            available_balance,
         } = value;
         let account_nonce = sequence_number.require()?.into();
         let account_amount = amount.require()?.into();
-        let account_release_schedule = schedule.require()?.try_into()?;
+        let account_release_schedule: AccountReleaseSchedule = schedule.require()?.try_into()?;
         let account_threshold = threshold.require()?.try_into()?;
         let account_encrypted_amount = encrypted_balance.require()?.try_into()?;
         let account_encryption_key = encryption_key.require()?.try_into()?;
         let account_index = index.require()?.into();
-        let account_stake = match stake {
+        let account_stake: Option<super::types::AccountStakingInfo> = match stake {
             Some(stake) => Some(stake.try_into()?),
             None => None,
         };
         let account_address = address.require()?.try_into()?;
+        let mut cds: Vec<super::types::Cooldown> = Vec::with_capacity(cooldowns.len());
+        for cooldown in cooldowns.into_iter() {
+            cds.push(cooldown.try_into()?)
+        }
+        let cooldowns = cds;
+
+        // The available balance is only provided as convenience and in case the
+        // calculation of it changes in the future. If the available balance is
+        // not present, we calculate it manually instead. Note that changes in the
+        // calculation method may make this calculation invalid.
+        let available_balance = available_balance.map(|ab| ab.into()).unwrap_or_else(|| {
+            let active_stake = account_stake.as_ref()
+                .map(|s| s.staked_amount())
+                // FIXME: Give common::types::Amount a `Default` impl and use .unwrap_or_default();
```

Comment:
I'm not sure we should do this, but if you think we should then either do it or create an issue for it.

Reply:
I think it makes sense since it's just a number. It makes sense to have a zero default I feel. I didn't do it here as it needs to be in [concordium_contracts_common](https://github.com/Concordium/concordium-base/tree/main/smart-contracts/contracts-common) and I'd need to do a new release of that crate for that then, which seemed a bit much just for this line. But I'll make an issue for it.

Reply:
https://github.com/Concordium/concordium-base/pull/538